ksqldb-engine/src/main/java/io/confluent/ksql/schema/registry/SchemaRegistryUtil.java

```diff
@@ -46,8 +46,10 @@ public final class SchemaRegistryUtil {
 
   @VisibleForTesting
   public static final int SUBJECT_NOT_FOUND_ERROR_CODE = 40401;
+  public static final int SCHEMA_NOT_FOUND_ERROR_CODE = 40403;
```

Comment:
Curious question how was the code `40403` determined?

Reply:
The error codes are from the SR code repo